src/actions/Chapter.ts

```diff
@@ -16,3 +16,13 @@ export async function chapterUpdateChapter(
     },
   });
 }
+
+export async function adminDeleteChapter(chapterId: string, reason: string) {
+  return internalRequest({
+    url: chapterAPI + `/${chapterId}/chapter`,
```

Comment:
I think its more intuitive to keep the URLs for admin functionality to be /api/natlAdmin/...